doc/openapi-user-guide.md

```diff
@@ -303,3 +304,133 @@ oc create secret generic threescale-provider-account --from-literal=adminURL=htt
 * Default provider account in the same namespace 3scale deployment
 
 The operator will gather required credentials automatically for the default 3scale tenant (provider account) if 3scale installation is found in the same namespace as the custom resource.
+
+## AWS STS Support
+
+With the new support for STS authentication (Secure Token Service for  short-term, limited-privilege security credentials), 
+the secret generated by the Cloud Credential tooling looks differ from AIM (Identity and Access Management) secret.
```

Comment:
```suggestion
the secret generated by the Cloud Credential tooling looks differ from IAM (Identity and Access Management) secret.
```

Reply:
Done + STS section was moved from openapi-user-guide.md instead to operator-user-guide.md.